### ISSUE TITLE
Unexpected error running Liquibase: java.lang.IllegalStateException

### ISSUE THREAD

**Issue by lovecodeyou** [liquibase/liquibase-data#37]
![image](https://user-images.githubusercontent.com/38997988/177163715-52b70791-f645-4259-a3ab-57ca5ef36c16.png)

When I run  【liquibase data run --name=myrepos --env=POSTGRES_PASSWORD=secret --image=postgres】，The command line displays the following information

**Unexpected error running Liquibase: java.lang.IllegalStateException: No context is configured, run 'titan install' or 'titan context install' to configure titan
For more information, please use the --log-level flag**

What is the meaning of this error not understand

**Comment by lovecodeyou** [comment]
I'm following this video command

**https://www.youtube.com/watch?v=bCeBpQvPJ5s**

**Comment by lovecodeyou** [comment]
> It looks like needs to be run. Have you gone through all the steps in this document, especially step 3?`liquibse data install`
> 
> https://github.com/liquibase/liquibase-data/blob/main/beta/session-1.md#step-3-download-liquibase-data-extension-and-add-to-your-system-path

I thought all the orders were just on the YouTube video，Thank you。I guess I didn't install docker. I'll try this document link you gave me.

**Comment by mcred** [comment]
Reply here if you run into any more issue.

**Comment by lovecodeyou** [comment]
OS：Vmware's centos7 image

run  **liquibase data install**   Keep repeating the following message，Is that normal？

![image](https://user-images.githubusercontent.com/38997988/177240843-43d72de3-24d9-478a-b5df-ca1b36cdedd5.png)

**Comment by mcred** [comment]
This happens where there is not a pre-compiled ZFS module for your OS. What is your host OS? Also, are you running docker desktop? The short answer is that you are missing ZFS, but the longer answer of how to fix that requires some more information.

**Comment by lovecodeyou** [comment]
> **What is your host OS**

centos7

> **are you running docker desktop?**

no，Just the Docker engine
![image](https://user-images.githubusercontent.com/38997988/177245731-cb476ff6-0d0d-4067-b8f2-7256d13b745d.png)

**Comment by lovecodeyou** [comment]
if i  use liquibase-data ,  Must Docker desktop be installed?

**Comment by mcred** [comment]
@lovecodeyou No, docker desktop isn't required. It is the usual installation method that MacOS and Windows users will use and we precompile and support ZFS modules for it. The underlying docker containers for liquibase data come from the titan project and there are some details about support [here](https://titan-data.io/docs/version/0.5.4/lifecycle/docker.html#linux).

Basically, not all possible kernel variants have pre-compiled ZFS modules. There are a few ways to solve your particular issue: 
  * Open a Pull Request [here](https://github.com/titan-data/zfs-releases) for your kernel variant. 
  * Clone the [zfs-builder](https://github.com/titan-data/zfs-builder) repo and run the tools there to build the ZFS kernel module. 
  * [Install ZFS on your own](https://openzfs.github.io/openzfs-docs/Developer%20Resources/Building%20ZFS.html)
